### PR TITLE
docs: create how-to-open-a-pull-request.md in german

### PR DESCRIPTION
**Pull request by Plingooo**
Translate to german

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by raisedadead** [comment]
Hi @Plingooo same as your previous pull-request can you get this PR reviewed

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!-- do not translate this -->
+| [Read these guidelines in other languages](/docs/i18n-languages) |
+|-|
+<!-- do not translate this -->
+
+# So erstellst du eine Pull Request
+
+## Wie man einen guten Pull Request Titel vorbereitet:
+
+Wenn du eine Pull Request(PR) erstellst, verwende die folgende Thementabelle, um zu entscheiden, wie du deinen PR im folgenden Format benennen möchtest:
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
Wenn du eine Pull Request(PR) erstellst, verwende die folgende Tabelle, um zu entscheiden, wie du deine PR im folgenden Format benennen möchtest:
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Ein Beispiel ist `fix(learn): Fixed tests for the do...while loop challenge`.
+
+| Thema | Dokumentation |
+|---|---|
+| `learn`,`curriculum` | Für Pull-Requests, die Änderungen an den Herausforderungen des Curriculums vornehmen. |
+| `client` | Für Pull-Requests, die Änderungen an der Logik der Clientplattform oder der Benutzeroberfläche vornehmen. |
+| `guide` | Für Pull-Requests, die Änderungen an dem Leitfaden vornehmen. |
+| `docs` | Für Pull-Requests, die Änderungen an der Projektdokumentation vornehmen. |
+
+## Vorschlag für einen Pull-Request (PR)
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
## Vorschlag für eine Pull-Request (PR)
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Sobald die Änderungen bestätigt wurden, wirst du aufgefordert, eine Pull Request auf der GitHub-Seite deines Forks zu erstellen.
+
+    ![Image - Compare pull request prompt on GitHub](/docs/images/github/compare-pull-request-prompt.png)
+
+2. Standardmäßig sollten alle Pull-Requests gegen den Hauptrepo von freeCodeCamp, Zweig `master`, gerichtet sein.
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
2. Standardmäßig sollten alle Pull-Requests gegen den `master` Zweig des Hauptrepositories von freeCodeCamp, gerichtet sein.
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    Stelle sicher, dass dein Standard Fork auf freeCodeCamp/freeCodeCamp eingestellt ist, wenn du einen Pull-Request auslöst.
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
    Stelle sicher, dass dein `base fork:` auf freeCodeCamp/freeCodeCamp eingestellt ist, wenn du eine Pull-Request auslöst.
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+    ![Image - Comparing forks when making a pull request](/docs/images/github/comparing-forks-for-pull-request.png)
+
+3. Sende die Pull-Anfrage aus deinem Zweig an den `master`- Zweig von freeCodeCamp.
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
3. Sende die Pull-Request aus deinem Zweig an den `master` Zweig von freeCodeCamp.
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+4. Füge im Text deiner PR eine detailliertere Zusammenfassung der von dir vorgenommen Änderungen und den Grund dafür hinzu.
+
+    - Es wird dir eine Pull Request Vorlage angezeigt. Dies ist eine Checkliste, die du vor dem Öffnen des Pull-Auftrags hättest befolgen sollen.
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
    - Es wird dir eine Pull Request Vorlage angezeigt. Dies ist eine Checkliste, die du vor dem Öffnen der Pull-Request hättest befolgen sollen.
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+    - Fülle die Details aus, wie es dir passt. Diese Informationen werden überprüft und entschieden, ob deine Pull-Anfrage akzeptiert wird oder nicht.
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
    - Fülle die Details aus, wie es dir passt. Diese Informationen werden überprüft und entschieden, ob deine Pull-Request akzeptiert wird oder nicht.
```

File: docs/i18n-languages/german/how-to-open-a-pull-request.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+    - Wenn die PR einen bestehenden Bug/Fehler beheben soll, dann füge am Ende von
+      der Beschreibung deines Pull Requests, das Schlüsselwort `closes` und #xxxx an (wobei xxxx
```
Comment (Squizzi3) [inline docs/i18n-languages/german/how-to-open-a-pull-request.md]:
```suggestion
      der Beschreibung deiner Pull Request, das Schlüsselwort `closes` und #xxxx an (wobei xxxx
```

**Review by martin-gnodtke — CHANGES_REQUESTED** [review]
Some minor consistency changes to @Plingooo's translation. Well done!

**Comment by Plingooo** [comment]
oh, I must have been a little tired. Thanks for the improvements @Squizzi3

**Review by raisedadead — APPROVED** [review]

**Pull request merged**